CIScripts/Build.ps1

```diff
@@ -84,6 +86,29 @@ try {
     if ($SconsBuildMode -eq "debug") {
         Copy-DebugDlls -OutputPath $DllsOutputDir
     }
+
+    if (Test-Path Env:DOCKER_REGISTRY) {
+        $ContainersAttributes = @(
+            @{
+                Suffix = "vrouter"
+                Folders = @(
+                    $vRouterOutputDir,
+                    $AgentOutputDir,
+                    $NodemgrOutputDir
+                )
+            },
+            @{
+                Suffix = "docker-driver"
+                Folders = @(
+                    $DockerDriverOutputDir
+                )
+            }
+        )
```

Comment:
Instead of using a hashtable we could define a class `ContainerAttribute`, e.g. like this:

```powershell
class ContainerAttributes {
    [string] $Suffix;
    [string[]] $Folders;

    ContainerAttributes([string] $Suffix, [string[]] $Folders) {
        $this.Suffix = $Suffix;
        $this.Folders= $Folders;
    }
}
```

Array construction would look like this:

```powershell
$ContainersAttributes = @(
    [ContainerAttributes]::New("vrouter", @(
        $vRouterOutputDir,
        $AgentOutputDir,
        $NodemgrOutputDir
    )),
    [ContainerAttributes]::New("docker-driver", @(
        $DockerDriverOutputDir
    ))
)
```